packages/office-ui-fabric-react/src/components/ProgressIndicator/ProgressIndicator.Props.ts

```diff
@@ -30,6 +30,11 @@ export interface IProgressIndicatorProps {
   percentComplete?: number;
 
   /**
+   * Text alternative of progress status
```

Comment:
> you should be clear in the comment that that's the only scenario in which is should be used.

@dzearing, in general users are cautious of setting aria attributes as this directly affects the screen readers experience. I have updated the comment but still leave out the caveat and scenario description as it will make description too long. We follow the same for other aria attributes as well.